datasets/nz/designated_terrorists/nz_designated_terrorists.yaml

```diff
@@ -0,0 +1,26 @@
+title: New Zeland's Designated Terrorist Entities
+entry_point: crawler.py
+prefix: nz-des-ter
+coverage:
+  frequency: daily
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  Individuals and entities that have been designated terrorists by the New Zeland government
+description: |
+  This dataset comprises individuals and entities that have been 
+  designated terrorist by the Terrorism Suppression Act 2002 (TSA).
+  The data comprises two different lists:
+  entities associated with Resolutions 1267/1989/2253/1988 and
+  entities associated with Resolution 1373. The former is the same
+  as the UNSC sanction list, so this dataset focus only on resolution 1373.
```

Comment:
Actually this is (correctly) only "non-UN listed entities designated in support of UN Security Council Resolution 1373"

It's good that we're not also re-listing all the entities already listed multiple times from the UN SC